PracticeMode.cs

```diff
@@ -493,22 +493,22 @@ private void HandleLoadNadeCommand(CCSPlayerController? player, string loadNadeN
                                 switch (lineupInfo["Type"])
                                 {
                                     case "Flash":
-                                        NativeAPI.IssueClientCommand((int)player.EntityIndex!.Value.Value - 1, "slot7");
+                                        NativeAPI.IssueClientCommand(player.Slot, "slot7");
```

Comment:
Hey!

I think we can directly use `player.ExecuteClientCommand("slotx");` now

Reply:
fixed!